MoonDawgArduino/Libraries/PinDefinitions/PinDefinitions.properties

```diff
@@ -0,0 +1,17 @@
+name=PinDefinitionsMoonDawg
+
+version=0.1.0
+
+author=Cameron Schwartzberg
+
+maintainer=camhotwheels@gmail.com
+
+sentence=This contains all of the pin definitions for the MoonDawgArduinoCode.ino.
+
+paragraph=To be added later.

```

Comment:
should this be added

Reply:
?

Reply:
Yes it's part of the .properties file which is used when you zip a library for the Arduino IDE

Reply:
The line silly goose, paragraph says “to be added later”

Reply:
oh lol, I dont feel like writing alat for something so small so I just made it "N/A" that way if I feel like it we can later